packages/simple-schema/lib/content-assist/element-name.js

```diff
@@ -12,9 +12,9 @@ const { difference, map, filter } = require("lodash");
  */
 function elementNameCompletion(elementNode, xssElement, prefix = "") {
   const allPossibleSuggestions = map(xssElement.elements, _ => _.name);
-  const notSingularElemNames = filter(
-    xssElement.elements,
-    _ => _.cardinality === "many"
+  const notSingularElemNames = map(

```

Comment:
evaluate these two high order functions as two **separate** steps with distinct variable names.

It makes it easier to debug.

See: https://youtu.be/kkTFx3-duc8?t=923